### PR TITLE
fix: add razaahmad333

### LINKED ISSUE #1285: Ahmad Raza
## Profile Screenshot

Screenshot (151)

### PR DESCRIPTION
**Pull request by razaahmad333**
resolves #1285 

## Checklist for profile addition

- [x] Name the PR `fix: add <your-username>`.
- [x] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [x] You've linked the PR to the correct issue.

---

## Other additions/changes

- [ ] Changes made in this PR are mentioned below
- [ ] Search previous suggestions before making a new one, as yours may be a duplicate.
- [ ] I acknowledge that all my contributions will be made under the project's license.
- [ ] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: profiles/razaahmad333.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "image":"https://user-images.githubusercontent.com/50910798/183011422-78c51113-785f-4a46-91ca-573693de64bf.png",
+    "issueId":1285,
+    "name":"Ahmad Raza",
+    "username":"razaahmad333"   
+}
```
Comment (Panquesito7) [inline profiles/razaahmad333.json:6]: Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient

File: profiles/razaahmad333.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+    "image":"https://user-images.githubusercontent.com/50910798/183011422-78c51113-785f-4a46-91ca-573693de64bf.png",
+    "issueId":1285,
+    "name":"Ahmad Raza",
+    "username":"razaahmad333"
```
Comment (Panquesito7) [inline profiles/razaahmad333.json:5]: Looks like there's a 4-space indentation here. We use a 2-space indentation.
```suggestion
  "image":"https://user-images.githubusercontent.com/50910798/183011422-78c51113-785f-4a46-91ca-573693de64bf.png",
  "issueId":1285,
  "name":"Ahmad Raza",
  "username":"razaahmad333"   
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: profiles/razaahmad333.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
+  "image":"https://user-images.githubusercontent.com/50910798/183011422-78c51113-785f-4a46-91ca-573693de64bf.png",
+  "issueId":1285,
+  "name":"Ahmad Raza",
+  "username":"razaahmad333"   
+}
+
```
Comment (Panquesito7) [inline profiles/razaahmad333.json:7]: There's an extra line here. GitHub doesn't display the endline.
```suggestion
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge!

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

File: profiles/razaahmad333.json [hunk 4]
```diff
@@ -0,0 +1,6 @@
+{
+  "image":"https://user-images.githubusercontent.com/50910798/183011422-78c51113-785f-4a46-91ca-573693de64bf.png",
+  "issueId":1285,
+  "name":"Ahmad Raza",
+  "username":"razaahmad333"
```
Comment (Panquesito7) [inline profiles/razaahmad333.json:5]: I think we need to add these extra spaces here.
```suggestion
  "image": "https://user-images.githubusercontent.com/50910798/183011422-78c51113-785f-4a46-91ca-573693de64bf.png",
  "issueId" :1285,
  "name": "Ahmad Raza",
  "username": "razaahmad333"   
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: profiles/razaahmad333.json [hunk 5]
```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/50910798/183011422-78c51113-785f-4a46-91ca-573693de64bf.png",
+  "issueId": 1285,
+  "name": "Ahmad Raza",
+  "username": "razaahmad333"
```
Comment (Panquesito7) [inline profiles/razaahmad333.json:5]: Trailing whitespace.
```suggestion
  "username": "razaahmad333"
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — APPROVED** [review]
Thanks for contributing! 🎉


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant

**Comment by razaahmad333** [comment]
> Thanks for contributing! 🎉
> 
> __ [sema-logo](https://semasoftware.com/gh) [ sema-logo ](https://semasoftware.com/gh) [ ](https://semasoftware.com/gh)  **Summary:** 👌 This code looks good  |  **Tags:** Elegant

its my pleasure

**Pull request merged**